Functions/Import/xASL_imp_NII2BIDS_RunPerf.m

```diff
@@ -60,11 +60,30 @@ function xASL_imp_NII2BIDS_RunPerf(imPar, bidsPar, studyPar, subjectSessionLabel
     end
     if xASL_exist(fullfile(inSessionPath, [aslLabel '.nii']),'file')
         headerASL = xASL_io_ReadNifti(fullfile(inSessionPath, [aslLabel '.nii']));
```

Comment:
What is "Label" here? Can we use the BIDS nomenclature? E.g. `subjectSessionValue` or `subjectSessionKey`? Can we replace "aslLabel" by what it is (e.g., `ASL4D_legacyLabel`, or `asl_FileName`). Now I have to open the actual code to understand what this means (though it probably means the FileName?

Reply:
`label` IS a BIDS nomenclature :)
But `aslLabel` was indeed a reference to the legacy name. So better to use legacy and not label.